packages/react-charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -673,10 +673,6 @@ export class LineChartBase extends React.Component<ILineChartProps, ILineChartSt
               onMouseOut={this._handleMouseOut}
               {...this._getClickHandler(this._points[i].onLineClick)}
               opacity={1}
-              role="img"
-              aria-label={`${legendVal}, series ${i + 1} of ${this._points.length} with ${
-                this._points[i].data.length
```

Comment:
why removed these labels

Reply:
Moved this aria-label to the wrapping `<g>` element